kubectl/clusterrolebinding.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
+metadata:
+  name: default-admin
```

Comment:
This seems like dangerous default behavior to recommend to users.

This grants the `default` SA (which runs every Pod in the cluster unless otherwise specified), _complete `cluster-admin` access_.

Can we either:

1. Make it clearer to users that this is only intended forr insecure/non-critical use cases, and should be modified after installation to match the user's use case and needs, and/or
2. Provide a more scoped-down role binding initially, which users can add to as they need?

Reply:
@ImJasonH 
Thanks for the comments, it's make sense, I will work for a pr for it.